build.gradle.kts

```diff
@@ -232,4 +235,58 @@ openApiMerger {
             }
         }
     }
-}
\ No newline at end of file
+}
+
+// Dependency analysis active if property "dependency.analysis" is set. Possible values are <'fail'|'warn'|'ignore'>.g po
```

Comment:
What's `.g po` in the end of the line?

Reply:
"g po" is my shortcut for "git push" 😂 thanks for spotting this one